### PR TITLE
Pre-package Playbooks `v1.32.1`

### PR DESCRIPTION
**Pull request by Phrynobatrachus**
Polish items for the redesigned Run Details and Playbook/Run list pages.

```release-note
Pre-package Playbooks v1.32.1
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -154,7 +154,7 @@ PLUGIN_PACKAGES += mattermost-plugin-channel-export-v1.0.0
 PLUGIN_PACKAGES += mattermost-plugin-custom-attributes-v1.3.0
 PLUGIN_PACKAGES += mattermost-plugin-github-v2.0.1
 PLUGIN_PACKAGES += mattermost-plugin-gitlab-v1.3.0
-PLUGIN_PACKAGES += mattermost-plugin-playbooks-v1.31.0
+PLUGIN_PACKAGES += mattermost-plugin-playbooks-v1.32.0
```
Comment (Phrynobatrachus) [inline Makefile:157]:
```suggestion
PLUGIN_PACKAGES += mattermost-plugin-playbooks-v1.32.1
```

**Review by gbochora — APPROVED** [review]
Some minor bugs, but overall looks good. Thanks, @Phrynobatrachus

**Pull request merged**